src/queue.js

```diff
@@ -253,15 +254,19 @@ function queue(config, emitter, store, storePlugin, breaker) {
    * @param {*} results The query results
    */
   function complete(key, ids, context, results) {
+    function contextRecordKey(id) {
```

Comment:
To prevent excess reference to variable and help GC, move `contextRecordKey` outside of this "class".
Should also help v8 optimise that function.

```javascript
const contextRecord = (key) => (id) => recordKey(key, id);
// ... code
function complete(key, ids, context, results) {
// ... more code
targetStore.set(contextRecord(key), ids, records, { step: 0 });
```

Reply:
Yeah sounds good